### ISSUE TITLE
No Save request after Shift Register Outputs modifications

### ISSUE THREAD

**Issue by zorzini** [MobiFlight/MobiFlight-Connector#606]
**Describe the bug**
Changing in "Select Pins" value in **Output** > **Display** > **Select Pins** for an **existing** Shift Register Mobiflight does not require to Save the changed configuration.

**Steps To Reproduce**
1. Go to 'Output >> Display >> Select Pins >>' then change the selected Output Led
2. Click on 'OK'

**Expected behavior**
"Save" button active if an item is changed in Output Settings.

**Actual behavior**
"Save" button not active after Output Setting modification.

**Details**
 - MobiFlight Version: 9.1.0.5
 - FlightSim: MSFS2020
 - AddOn: any

**Additional context**
The test carries out the new sequence but closing and restarting MF the change is lost.
It is necessary to remember File >> Save As ... as the "Save" button is not active.
It seems that is a cache for this applied modifications but the *.mcc file is not modified and also no Save will be request.

![output 01](https://user-images.githubusercontent.com/65813913/144758546-87b5902c-d473-4987-8796-5afe4f0154f5.jpg)
![output 02](https://user-images.githubusercontent.com/65813913/144758545-dce15e4a-7b0f-467c-9491-2921c3651371.jpg)
![output 03](https://user-images.githubusercontent.com/65813913/144758543-430988e6-79d4-4f40-91b8-37cf3901d674.jpg)

**Comment by MobiFlight-Admin** [comment]
I found the source of this error being the missing equals implementation. I have fixed this locally on my branch.
However, I noticed that the save button now would also turn active when no changes are made because of two problems:

* wrong intialization of SimConnect Variable settings, which lead to the fact that even if we dont make changes to the SimConnect Variable it still changes
* the LED Address for a module changes even though we dont have an LED module defined.
